packages/devtools_app/lib/src/screens/profiler/cpu_profiler.dart

```diff
@@ -365,224 +367,62 @@ class _CpuProfilerState extends State<CpuProfiler>
   }
 }
 
-class DisplayTreeGuidelinesToggle extends StatelessWidget {
```

Comment:
all the deletes over here were moved to profiler_controls.dart with no functional changes. git diff picked this up weird unfortunately.